softlayer/vm/softlayer_hardware.go

```diff
@@ -148,6 +148,8 @@ func (vm *softLayerHardware) ConfigureNetworks(networks Networks) error {
 		return bosherr.WrapErrorf(err, "Failed to unmarshal userdata from hardware with id: %d.", vm.ID())
 	}
 
+	bslcommon.RETRY_COUNT = 30
```

Comment:
The reason I was asking to put `RETRY_COUNT` and `WAIT_TIME` in the `common` package was to avoid having to set these values every time. Why can this be set once (default or the value in environment variables) and used?